### DISCUSSION TITLE
detach_window new-tab with neighbor window location

### DISCUSSION THREAD

**Discussion by MinmoTech** [kovidgoyal/kitty#5770] [General]
Hey! :)

I'd like to detach a window into a new tab but for that tab to appear right to the current tab (in the neighbor position)
With new tabs in general this is possible with
`map kitty_mod+t     launch --cwd=current --type=tab --location=neighbor`

Is the same possible while detaching a window?
This would enable one to quickly reattach to the previous tab with
`map kitty_mod+alt+h detach_window tab-left`

**Comment by kovidgoyal** [comment]
No, detach_window by itself cannot do that. If you want to re-attach to
the previous tab just do detach_window tab-prev which will work unless
you have switched to a third tab in the meantime.

**Reply by MinmoTech** [reply]
Thanks!
